api/v1alpha2/terraform_types.go

```diff
@@ -479,6 +479,14 @@ type TFStateSpec struct {
 	//
 	// +optional
 	LockIdentifier string `json:"lockIdentifier,omitempty"`
+
+	// LockTimeout is a Duration string that instructs Terraform to retry acquiring a lock for the specified period of
+	// time before returning an error. The duration syntax is a number followed by a time unit letter, such as `3s` for
+	// three seconds.
```

Comment:
[not a blocker]
Can we add a note about the `0s` value, I think it's not obvious what happens if the timeout is `0s`.

Reply:
Added extra info :+1: